tests/php/unit/Configuration/ConfigurationTest.php

```diff
@@ -71,14 +72,53 @@ public function testBlacklistFilterInvalid() {
 			new \Yoast_ACF_Analysis_String_Store()
 		);
 
-		Filters\expectApplied( \Yoast_ACF_Analysis_Facade::get_filter_name( 'blacklist' ) )
+		Filters\expectApplied( \Yoast_ACF_Analysis_Facade::get_filter_name( 'blacklist_type' ) )
 			->once()
 			->with( $store )
 			->andReturn( '' );
 
-		$this->assertSame( $store, $configuration->get_blacklist() );
+		$this->assertSame( $store, $configuration->get_blacklist_type() );
+	}
+
+	public function testBlacklistNameFilter() {
+
+		$blacklist_name = new \Yoast_ACF_Analysis_String_Store();
+
+		$configuration = new \Yoast_ACF_Analysis_Configuration(
+			$blacklist_name,
```

Comment:
The first parameter passed to the constructor of `Yoast_ACF_Analysis_Configuration` is the store for `blacklist_type` and not `blacklist_name`. As this is not what we're testing this should be
`new \Yoast_ACF_Analysis_String_Store();`